docs/core/deploying/index.md

```diff
@@ -13,17 +13,17 @@ ms.assetid: da7a31a0-8072-4f23-82aa-8a19184cb701
 
 # .NET Core Application Deployment
 
-You can create two types of deployments for .NET Core applications: 
+You can create two types of deployments for .NET Core applications:
 
 - Framework-dependent deployment. As the name implies, framework-dependent deployment (FDD) relies on a shared system-wide version of .NET Core to be present on the target system. Because .NET Core is already present, your app is also portable between installations of .NET Core. Your app contains only its own code and any third-party dependencies that are outside of the .NET Core libraries. FDDs contain .dll files that can be launched by using the [dotnet utility](../tools/dotnet.md) from the command line. For example, `dotnet app.dll` runs an application named `app`.
 
 - Self-contained deployment. Unlike FDD, a self-contained deployment (SCD) does not rely on any shared components to be present on the target system. All components, including both .NET Core libraries and the .NET Core runtime, are included with the application and are isolated from other .NET Core applications. SCDs include an executable (such as `app.exe` on Windows platforms for an application named `app`), which is  a renamed version of the platform-specific .NET Core host, and a .dll file (such as `app.dll`), which is the actual application.
 
-## Framework-dependent deployments (FDD) ##
+## Framework-dependent deployments (FDD)
 
 For an FDD, you deploy only your app and any third-party dependencies. You do not have to deploy .NET Core, since your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core apps.
 
-### Why create a framework-dependent deployment? ###
+### Why create a framework-dependent deployment?
 
 Deploying an FDD has a number of advantages:
```

Comment:
In the list below ...

First bullet:
* "do not" :point_right: "don't"
* "You do not have to define the target operating systems that your .NET Core app will run on in advance." :point_right: "You don't need to define in advance the target operating systems where your .NET Core app will run."
* "PE file format" isn't defined (it's linked, but not defined)
* The 2nd sentence somewhat restates the info in the first sentence

Second bullet:
* "You only have to deploy" :point_right: "You only deploy"